inc/template.php

```diff
@@ -341,6 +341,7 @@ function tpl_metaheaders($alt = true) {
         $script .= "var SIG='".toolbar_signature()."';";
     }
     jsinfo();
+    $JSINFO['conf']['mediarevisions'] = $conf['mediarevisions'];
```

Comment:
This line should definitely go to `inc/common.php` if needed. I am trying to see the best practice to bring this to the specific page that requires this.